src/utils/structuredData.ts

```diff
@@ -0,0 +1,75 @@
+import { AccordionProps } from 'blocks/AccordionBlock/AccordionBlock'
+
+export const structuredFAQPage = (
+  accordions: ReadonlyArray<AccordionProps>,
+) => ({
+  '@context': 'http://schema.org',
+  '@type': 'FAQPage',
+  mainEntity: accordions.map((item) => ({
+    '@type': 'Question',
+    name: item.title,
+    acceptedAnswer: {
+      '@type': 'Answer',
+      text: item.paragraph.html,
+    },
+  })),
+})
+
+export const structuredSoftwareApplication = () => [
+  {
+    '@context': 'http://schema.org',
+    '@type': 'SoftwareApplication',
+    name: 'Hedvig',
+    operatingSystem: 'IOS',
+    applicationCategory: 'FinanceApplication',
+    aggregateRating: {
+      '@type': 'AggregateRating',
+      ratingValue: '4.7',
+      ratingCount: '1200',
```

Comment:
Is there a way to not hard code these? Feels like something we most definitely will forget to update so it will pretty soon be no longer up-to-date?

Reply:
Yeah for sure. It's kind of arbitrary that we are allowed to set the values ourselves.

But I'm okey with removing these as well for this PR if we want to investigate it some more!

Reply:
Hm, yeah I would say we could add these later if we can find a way to get those ratings directly from App Store/Google Play